pkgs/applications/graphics/grafx2/default.nix

```diff
@@ -37,12 +37,16 @@ stdenv.mkDerivation rec {
   makeFlags = [ "-C src" ];
   installFlags = [ "-C src" "PREFIX=$(out)" ];
 
+  postInstall = ''
+    mv $out/bin/grafx2-sdl $out/bin/grafx2
+  '';
```

Comment:
this should probably be a symlink so it doesn't break anything using the old binary name.